assets/src/edit-story/app/font/actions/useLoadFontFiles.js

```diff
@@ -19,45 +19,72 @@
  */
 import { useCallback } from 'react';
 
+/**
+ * Internal dependencies
+ */
+import { MULTIPLE_VALUE } from '../../../components/form';
+
 function useLoadFontFiles({ getFontByName }) {
   /**
    * Adds a <link> element to the <head> for a given font in case there is none yet.
    *
    * Allows dynamically enqueuing font styles when needed.
    *
-   * @param {string} name Font name.
+   * @param {string} fontFamily Font name.
    */
   const maybeEnqueueFontStyle = useCallback(
-    (name) => {
-      if (!name) {
-        return;
-      }
+    ({ fontFamily, fontWeight, fontStyle, content }) => {
+      return new Promise((resolve, reject) => {
+        // skip when required infos are not declared
+        if (!fontFamily || !fontWeight || !fontStyle) {
+          return null;
+        }
+        const fontFaceSet = `${fontStyle} ${fontWeight} 0 '${fontFamily}'`;
+        const hasMultiple = fontFaceSet.indexOf(MULTIPLE_VALUE) > -1;
```

Comment:
@dvoytenko can you please check this proposal for MULTIPLE special case considering the FontFaceSet API for `.check`/`.load`, which require explicitly `"font-style font-weight font-size 'font-family'"`? My approach just ignore preload the font (which don't apply the fix properly to resize box) when the FontFaceSet is invalid, but if you have another better proposal, I would love to hear, maybe add default values when some of the variants are `((MULTIPLE))`?